doc/notebooks/jupyter_integration.ipynb

```diff
@@ -79,7 +79,7 @@
    "outputs": [],
    "source": [
     "# Start GRASS Session\n",
-    "gj.init(\"/home/data/grassdata\", \"nc_basic_spm_grass7\", \"user1\")\n",
+    "gj.init(\"../../data/grassdata", \"nc_basic_spm_grass7\", \"user1\")\n",
```

Comment:
The red in the `+` line marks the error in the JSON syntax which is what is causing the following message when you open the notebook:

```
File Load Error for jupyter_integration.ipynb
Unreadable Notebook: /home/jovyan/doc/notebooks/jupyter_integration.ipynb NotJSONError('Notebook does not appear to be JSON: \'{\\n "cells": [\\n {\\n "cell_type": "m...')
```

The fix is putting back the `\` for the double quote after `grassdata` which should cause the red-marked `\` to be interpreted correctly.